python_modules/libraries/dagster-components/dagster_components/core/python_env_components.py

```diff
@@ -0,0 +1,154 @@
+import importlib
```

Comment:
This file is very scary and is a persuasive argument to not do this. I think we will have to just bite the bullet and do conditional imports.